vscode-extensions/process-editor/extension/package.json

```diff
@@ -32,9 +32,6 @@
   "engines": {
     "vscode": "^1.54.0"
   },
-  "activationEvents": [
-    "onCommand:vscode-process-editor-extension.activate"
```

Comment:
no activation needed. process-editor gets activated the first time a *.p.json file is opened.